cliquet/resource.py

```diff
@@ -753,6 +755,15 @@ def _add_timestamp_header(self, response, timestamp=None):
         # Return timestamp as ETag.
         response.headers['ETag'] = encode_header('"%s"' % timestamp)
 
+    def _add_cache_header(self, response):
+        """Add Cache-Control and Expire headers, based a setting for this
+        resource.
+        """
+        key = 'cliquet.%s_cache_expires_seconds' % self.context.resource_name
```

Comment:
I think we can leave it like that for now, since it matches our use case and then add more precises settings if needed.

For now, I think we should have the same for collection_get and get, also we may want to differenciate between authenticated and unauthenticated (cache for unauthenticated only for instance)